### DISCUSSION TITLE
[p216] CORS 질문드립니다.

### DISCUSSION THREAD

**Discussion by HUIWON-RHO** [fsoftwareengineer/todo-application#83] [질문과 답변] [answered]
![image](https://user-images.githubusercontent.com/59330544/154784123-bf0b19bc-ece8-4e9c-adea-87a7ec0f5ecc.png)
![image](https://user-images.githubusercontent.com/59330544/154784141-5889a2e8-a70c-45c2-85e1-779f45f7bb7f.png)

실습코드 3-40의 WebMvcConfig를 따라 작성하였는데 화면과 같은 에러가 계속 발생합니다.
구글링하여 혼자 해결해보려고 했지만 도저히 해결되지 않아 질문드립니다.

혹시나 발견하지 못하는 오타가 있는가 싶어 작가님이 올려주신 코드를 그대로 복붙해 봤는데도 안되네요... 도와주시면 정말 감사하겠습니다!

**Comment by gihohpkl12** [comment]
작가님이 아니라서 정확하진 않지만
프론트에서 에러가 아니라면 bean등록이 안 된 것 같은데 
사진에서는 @SpringBootApplication 위치가 안 나와서..
@ComponentScan 위치는 확인해보셨나요??

**Reply by HUIWON-RHO** [reply]
답변 정말 감사합니다! 고민해볼만한 의견이었습니다!!

**Comment by fsoftwareengineer** [comment, marked as answer]
Application.java 또는 메인메서드가 있는 클래스는 어디에 있나요? com.example.demo아래에 있어야 합니다. 스프링은 메인이 있는 패키지의 하위패키지를 검색해 컴포넌트를 등록하므로 혹시 다른 패키지에 메인이 있어 그런것이 아닌지 확인 해 주시기 바랍니다.

**Reply by HUIWON-RHO** [reply]
답변 감사합니다, 작가님. 어떻게 메인메소드가 없는데 에러가 안나고 실행이 되었는지 의문이네요...뭘하다가 메인메소드를 지워버린건지도 의문이구요...ㅋㅋ 계속 CORS문제겠거니 했는데... 정말 감사합니다!
